SPECS/fsverity-utils/fsverity-utils.spec

```diff
@@ -0,0 +1,116 @@
+Summary:        fsverity utilities
+Name:           fsverity-utils
+Version:        1.4
+Release:        8%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/
+Source0:        https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/snapshot/%{name}-%{version}.tar.gz
+Patch1:         %{name}-pkcs11-opaque-keys-support.patch
+BuildRequires:  gcc
+BuildRequires:  glibc-headers
+BuildRequires:  kernel-headers
+BuildRequires:  make
+BuildRequires:  openssl-devel
+Requires:       libfsverity = %{version}-%{release}
+
+%description
+This is fsverity, a userspace utility for fs-verity.
+fs-verity is a Linux kernel feature that does transparent on-demand
+integrity/authenticity verification of the contents of read-only files,
+using a hidden Merkle tree (hash tree) associated with the file.
+The mechanism is similar to dm-verity, but implemented at the file level
+rather than at the block device level. The fsverity utility allows you
+to set up fs-verity protected files.
+
+%package -n libfsverity
+Summary:        Development package for fsverity-utils
+
+%description -n libfsverity
+Library for fsverity-utils.
+
+%package devel
+Summary:        Development package for fsverity-utils
+Requires:       %{name} = %{version}-%{release}
+Requires:       libfsverity = %{version}-%{release}
+
+%description devel
+Development package for fsverity-utils. This package includes the
+libfsverity header and library files.
+
+%prep
+%autosetup -p1
+
+%build
+%{set_build_flags}
+%make_build CFLAGS="$CFLAGS -g" USE_SHARED_LIB=1
+
+%install
+%{set_build_flags}
+%make_install PREFIX=%{_prefix} LIBDIR=%{_libdir}  CFLAGS="$CFLAGS -g" USE_SHARED_LIB=1
+find %{buildroot} -type f -name "*.a" -delete
+
+%files
+%doc README.md
+%{_bindir}/fsverity
+
+%files -n libfsverity
+%license LICENSE
+%{_libdir}/libfsverity.so.0
+
+%files devel
+%{_includedir}/libfsverity.h
+%{_libdir}/libfsverity.so
+%{_libdir}/pkgconfig/libfsverity.pc
+
+%changelog
+* Thu Apr 20 2023 Elaheh Dehghani <edehghani@microsoft.com> - 1.4-8
+- Initial overlake-packages import from Fedora 36 (license: MIT).
```

Comment:
Can we change this to mention Mariner instead?

Reply:
Also, updating the date to be relevant to when it was introduced to this repo would be nice.

Reply:
Yes, please update the comment to `Initial CBL-Mariner import from Fedora 36 (license: MIT).`. This is required for the spec source attribution check.

Related to @oliviacrain's comment around the `License` tag: to resolve the PR's license check, please also verify that the license declared in the spec's `License` tag matches the actual license of the project. I see that there is a `LICENSE` file in the package already, so that would be a good starting point. Please use the [SPDX license identifiers](https://spdx.org/licenses/) in the spec's `License` tag. It sounds like most likely the license is MIT.

Once you're sure you've verified the license correctly you should indicate that by adding the `License verified.` line to the changelog - this will unblock the PR check.

Reply:
Thanks! updated changelog